### ISSUE TITLE
supports_extra_name_on_order

### ISSUE THREAD

**Issue by babybunny** [babybunny/rebuildingtogethercaptain#278]
implement supports_extra_name_on_order

don't recall what it does ... something about being able to add a name to an orderitem.

**Comment by caripang** [comment]
unfortunately, i'm not sure what that is either.  perhaps it's the delivery/pickup contact info for a particular order?

**Comment by babybunny** [comment]
Researching.  

There was a per-item name on the order form where you add quantities for each item.  It was editable and optional. 

In the database there are values like "#23 Swiss Coffee"  so I think it's used for paint colors.  
There are also values like "Adam" which looks like the way staff time was handled before we implemented a special stafftime object.

it applies to these items
https://console.cloud.google.com/datastore/entities/query?project=rebuildingtogethercaptain-hrd&ns=&kind=Item&filter=28%2Fsupports_extra_name_on_order%7CBL%7CEQ%7C4%2Ftrue

and all the items on these order forms
https://console.cloud.google.com/datastore/entities/query?project=rebuildingtogethercaptain-hrd&ns=&kind=OrderSheet&filter=28%2Fsupports_extra_name_on_order%7CBL%7CEQ%7C4%2Ftrue


reference: from old order_layout.html 
        {% if order_item.SupportsName %}
              -
              <span class="jeditable-order-item-name" 
                    id="{{ order_item.key.id }}"
                    >{{ order_item.name }}</span>
              {% endif %}

**Comment by caripang** [comment]
Yes - it looks like paint orders, t-shirt orders, and staff time.  

The SMV (Safety Mountain View) form was recently created and quickly abandoned so that doesn't need to be brought over to the new system.

The HHC (Healthy Housing Challenge) form also has not been used much, so may not be needed anymore as I think most of the items ended up being moved to the Safety Materials form (separate from SMV) and we're no longer tracking items separately for the HHC (an initiative that has since been completed).

The Runners Order Form is not something that we've been using recently as it was never really used much by the Captains.

**Comment by babybunny** [comment]
can we drop this feature?
